router/src/main/kotlin/io/moia/router/Router.kt

```diff
@@ -78,6 +78,7 @@ data class Request<I>(val apiRequest: APIGatewayProxyRequestEvent, val body: I,
     val pathParameters by lazy { UriTemplate.from(pathPattern).extract(apiRequest.path) }
     val queryParameters: Map<String, String>? by lazy { apiRequest.queryStringParameters }
     val multiValueQueryStringParameters: Map<String, List<String>>? by lazy { apiRequest.multiValueQueryStringParameters }
+    val requestContext by lazy { apiRequest.requestContext }
```

Comment:
makes sense! The only thing is  that its type is "inferred from plattform call".  I would use `val requestContext: APIGatewayProxyRequestEvent.ProxyRequestContext by lazy { apiRequest.requestContext }` to get rid of warnings inside IntelliJ.

Reply:
@jmoennich done did it 👍